sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md

```diff
@@ -1,20 +1,10 @@
 # Release History
 
-## 1.6.0 (2025-05-01)
+## 1.5.0 (2025-05-01)
```

Comment:
You'll want to update the date here to represent when this is going to be shipped (today)